app/design/adminhtml/default/default/template/ebanx/info/default.phtml

```diff
@@ -29,7 +29,7 @@
       <button id="update-now">Check Now</button>
       <div id="update-error" style="color: red;"></div>
     </td>
-    <script type="text/javascript">
+    <script>
```

Comment:
Why removing the type? Also, why did you leave this script inline?

Reply:
I removed all types to leave they with the same default tag, but don't make sense do this. Also I leave this script inline, because I don't know when I will add the script file inside the ebanx.xml